blog/2023-10-20-hydra.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Hydra Team Update
+slug: 2023-10-20-hydra
+authors: [ffakenz]
+tags: [hydra]
+hide_table_of_contents: false
+---
+
+## High-level summary
+
+This week, the Hydra team primarily focused on preparing a masterclass and workshop for #CardanoSummit2023.
+
+In addition, they achieved the following milestones:
+
+- Enhanced the Hydra node API's submit-transaction endpoint to accept three different encoding types: Base16 encoded CBOR string, TextEnvelope type, and JSON. 
+  This improvement offers users greater flexibility and ease of interaction with the API.
+
+- The team has diligently followed up on the TUI brick upgrade to version 1.10. 
+  This effort has resulted in addressing minor details and enhancing the overall user experience with the TUI, making it more user-friendly and efficient.
+
+These accomplishments reflect the team's commitment to delivering excellence and ensuring a seamless experience for users and stakeholders.
```

Comment:
WTF? Are you generating those with ChatGPT or what?

Reply:
Will remove this phrasing.